app/lib/features/events/pages/event_details_page.dart

```diff
@@ -500,24 +493,33 @@ class _EventDetailPageConsumerState extends ConsumerState<EventDetailPage> {
   }
 
   Widget _buildEventDataSet(CalendarEvent ev) {
-    final inDays =
+    final agoTime =
         Jiffy.parseFromDateTime(toDartDatetime(ev.utcStart()).toLocal())
-            .endOf(Unit.day)
+            .endOf(Unit.hour)
             .fromNow();
     final startDate =
         Jiffy.parseFromDateTime(toDartDatetime(ev.utcStart()).toLocal())
-            .format(pattern: 'EEE, MMM dd AT hh:mm');
+            .format(pattern: 'EEE, MMM dd, yyyy AT hh:mm');
     final endDate =
         Jiffy.parseFromDateTime(toDartDatetime(ev.utcEnd()).toLocal())
-            .format(pattern: 'EEE, MMM dd AT hh:mm');
+            .format(pattern: 'EEE, MMM dd, yyyy AT hh:mm');
+
+    String eventTimingTitle = '';
+    if (getEventType(ev) == EventFilters.ongoing) {
+      eventTimingTitle = '${L10n.of(context).started} $agoTime';
+    } else if (getEventType(ev) == EventFilters.upcoming) {
+      eventTimingTitle = '${L10n.of(context).starts} $agoTime';
+    } else if (getEventType(ev) == EventFilters.past) {
+      eventTimingTitle = '${L10n.of(context).ended} $agoTime';
+    }
```

Comment:
this isn't the proper way to do this ... for two reasons: 1. "days" isn't international  and we aren't correctly setting jiffy i18n at the moment (reported at https://github.com/acterglobal/a3/issues/2010 ) and 2 the grammatical order might be different in other languages.

Lastly, let's be specific with our L10n keys please as the word itself might otherwise be too generic and confusing when used in other languages and the same key is used:

```suggestion
    if (getEventType(ev) == EventFilters.ongoing) {
      eventTimingTitle = L10n.of(context).eventStarted(agoTime);
    } else if (getEventType(ev) == EventFilters.upcoming) {
      eventTimingTitle = L10n.of(context).eventstarts(agoTime);
    } else if (getEventType(ev) == EventFilters.past) {
      eventTimingTitle = L10n.of(context).eventEnded(agoTime);
    }
```